src/libs/registry/docset.cpp

```diff
@@ -607,3 +617,183 @@ QString Docset::parseSymbolType(const QString &str)
 
     return aliases.value(str, str);
 }
+
+// ported from DevDocs' searcher.coffee:
+// (https://github.com/Thibaut/devdocs/blob/50f583246d5fbd92be7b71a50bfa56cf4e239c14/assets/javascripts/app/searcher.coffee#L91)
+static void matchFuzzy(
```

Comment:
Just put parameters on the same line, and break to fit the per line character limit.